packages/interpretations/src/components/CollapsibleCard.js

```diff
@@ -8,46 +8,7 @@ import CardContent from '@material-ui/core/CardContent';
 import Collapse from '@material-ui/core/Collapse';
```

Comment:
Move this file inside of the /details folder since that is where it is used.

Reply:
Done!